sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -99,7 +99,15 @@ class StreamExecution(
   /** The thread that runs the micro-batches of this stream. */
   private[sql] val microBatchThread =
     new UninterruptibleThread(s"stream execution thread for $name") {
-      override def run(): Unit = { runBatches() }
+
+      private val callSite = Utils.getCallSite()
```

Comment:
@zsxwing making it as a field of `StreamExecution` is surely more reasonable; I've updated this within the new commit. Thanks!